src/RpcClient/RpcClient.cs

```diff
@@ -199,34 +210,39 @@ public RpcRawMemPool GetRawMempoolBoth()
         /// <summary>
         /// Returns the corresponding transaction information, based on the specified hash value.
         /// </summary>
-        public string GetRawTransactionHex(string txid)
+        public string GetRawTransactionHex(string txHash)
         {
-            return RpcSend("getrawtransaction", txid).AsString();
+            return RpcSend("getrawtransaction", txHash).AsString();
         }
 
         /// <summary>
         /// Returns the corresponding transaction information, based on the specified hash value.
         /// verbose = true
         /// </summary>
-        public RpcTransaction GetRawTransaction(string txid)
+        public RpcTransaction GetRawTransaction(string txHash)
```

Comment:
vebose as argument?

Reply:
The return types are not the same, I think using two methods is more convenient.